### DISCUSSION TITLE
[suggestion] "Unlisted only" option for starred repos

### DISCUSSION THREAD

**Discussion by 1280px** [community/community#10499] [Other Feature Feedback, Questions, & Ideas]
Add a toggle-able option for starred repositories dashboard to only show stars that not added to any list.

![изображение1](https://user-images.githubusercontent.com/71165491/150834193-8f3a2150-ce36-4931-98c6-f7e9741a6073.png)

![изображение2](https://user-images.githubusercontent.com/71165491/150833889-abd138ee-ffd2-4bbc-a8a6-f2a087df9edf.png)

**Comment by movahhedi** [comment]
+1 Looking forward to this

**Comment by 743859910** [comment]
+1 Looking forward to this

**Comment by movahhedi** [comment]
Still waiting...

**Comment by benblank** [comment]
Because there [isn't even a way to see lists via an API](https://github.com/community/community/discussions/8293), you also can't really make a script or browser extension to do this without a lot of page scraping. The only "reasonable" way to see which of your stars haven't been added to a list is… manually. 😤

**Comment by Jibggjii** [comment]
> Add a toggle-able option for starred repositories dashboard to only show stars that not added to any list.
> 
> ![изображение1](https://user-images.githubusercontent.com/71165491/150834193-8f3a2150-ce36-4931-98c6-f7e9741a6073.png)
> 
> ![изображение2](https://user-images.githubusercontent.com/71165491/150833889-abd138ee-ffd2-4bbc-a8a6-f2a087df9edf.png)
>

**Comment by onesianinflaredguard** [comment]
Love you too you'Unityouright[]()you'highlight

**Comment by ksharizard** [comment]
Hi! I was facing the same issue with having a lot of stars and not knowing which are unlisted. I created a bash script to get a user's stars and lists and find the unlisted stars. I tested this on my profile and it worked well to output the unlisted repos. Just linking it here if anybody finds it interesting and wants to try it out:

https://github.com/ksharizard/github-lists
